lib/cli.js

```diff
@@ -83,6 +83,12 @@ function cli()
 				info: "URL to an HTML document.",
 				type: require("url")
 			},
+			"maxSocketsPerHost":
+			{
+				type: Number,
+				info: "The maximum number of links per host/port to check at any given time.",
```

Comment:
This string is too long for the help screen.

Reply:
made it shorter.

Reply:
I think "requests" is clearer than "checks".

Reply:
fixed